src/main/java/org/gradle/profiler/buildops/BuildOperationInstrumentation.java

```diff
@@ -66,7 +65,8 @@ private String newFile(File dataFile) {
         }
         try {
             try (Stream<String> lines = Files.lines(configurationTimeDataFile.toPath(), StandardCharsets.UTF_8)) {
-                return Streams.findLast(lines)
+                return lines
+                    .reduce((first, second) -> second)
```

Comment:
:( I hate that we have to make so much code worse because of our inability to upgrade Guava to the JRE version.